doc/api/http.md

```diff
@@ -2755,6 +2755,16 @@ http.get('http://nodejs.org/dist/index.json', (res) => {
 }).on('error', (e) => {
   console.error(`Got error: ${e.message}`);
 });
+
+// Create a local server to receive data from
+const server = http.createServer((req, res) => {
+  res.writeHead(200, { 'content-type': 'application/json' });
```

Comment:
The case is all over the place in the document, but I think it's "more correct" to capitalize the header field as it is in the spec https://tools.ietf.org/html/rfc7231#section-3.1.1.5
```suggestion
  res.writeHead(200, { 'Content-Type': 'application/json' });
```